src/main/java/uk/ac/cam/cl/dtg/isaac/dos/LightweightQuestionValidationResponse.java

```diff
@@ -92,9 +96,28 @@ public void setDateAttempted(final Date dateAttempted) {
         this.dateAttempted = dateAttempted;
     }
 
+    /**
+     * Gets the marks.
+     *
+     * @return the marks
+     */
+    public Integer getMarks() {
+        return marks;
+    }
+
+    /**
+     * Sets the marks.
+     *
+     * @param marks
+     *            the marks to set
+     */
+    public void setMarks(final Integer marks) {
+        this.marks = marks;
+    }
+
     @Override
     public String toString() {
         return "QuestionValidationResponse [questionId=" + questionId + ", correct=" + correct +
-                ", dateAttempted=" + dateAttempted + "]";
+                ", dateAttempted=" + dateAttempted + ", marks=" + marks + "]";
```

Comment:
It's really not important, but I'd have put the marks before the date, so that is was with the correct.